variables.tf

```diff
@@ -37,3 +37,9 @@ variable "enable_gcci_boundary" {
   type        = bool
   default     = true
 }
+
+variable "create_access_key" {
+  description = "Create user access key if enabled"
+  type        = bool
+  default     = true
```

Comment:
might need to give `pgp_key` a default of empty string so that if this is false, it can still create successfully without providing `pgp_key`

Reply:
tested false condition locally, still can create successfully without `pgp_key`. Prefer to not give default empty string for `pgp_key` so that it'll prompt if we forget to enter.

Reply:
oh cool